seeya_server/settings/common.py

```diff
@@ -127,7 +127,7 @@
             "bucket_name": os.getenv("SEEYA_S3_BUCKET_NAME"),
             "access_key": os.getenv("AWS_ACCESS_KEY_ID"),
             "secret_key": os.getenv("AWS_SECRET_ACCESS_KEY"),
-            "querystring_auth": False,
+            "querystring_auth": True,
```

Comment:
https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
> querystring_auth or AWS_QUERYSTRING_AUTH
> Default: True
> Setting AWS_QUERYSTRING_AUTH to False to remove query parameter authentication from generated URLs. This can be useful if your S3 buckets are public.
버킷이 public일때 사용하는 설정이라네요